test/evm/hardhat/chain-specific-spokepools/Base_SpokePool.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable node/no-missing-import */
```

Comment:
I don't really like disabling this, instead I'd rather just fix the missing import